x/funders/types/errors.go

```diff
@@ -6,13 +6,15 @@ import (
 
 // x/funders module sentinel errors
 var (
-	ErrFunderAlreadyExists      = errors.Register(ModuleName, 1100, "funder with address %v already exists")
-	ErrFunderDoesNotExist       = errors.Register(ModuleName, 1101, "funder with address %v does not exist")
-	ErrFundsTooLow              = errors.Register(ModuleName, 1102, "minimum funding amount of %vkyve not reached")
-	ErrAmountPerBundleTooLow    = errors.Register(ModuleName, 1103, "minimum amount per bundle of %vkyve not reached")
-	ErrMinFundingAmount         = errors.Register(ModuleName, 1104, "minimum funding amount of %vkyve not reached")
-	ErrFundingDoesNotExist      = errors.Register(ModuleName, 1105, "funding for pool %v and funder %v does not exist")
-	ErrFundingIsUsedUp          = errors.Register(ModuleName, 1106, "funding for pool %v and funder %v is used up")
-	ErrFundingStateDoesNotExist = errors.Register(ModuleName, 1107, "funding state for pool %v does not exist")
-	ErrMinFundingMultiple       = errors.Register(ModuleName, 1108, "per_bundle_amount (%dkyve) times min_funding_multiple (%d) is smaller than funded_amount (%vkyve)")
+	ErrFunderAlreadyExists               = errors.Register(ModuleName, 1100, "funder with address %v already exists")
+	ErrFunderDoesNotExist                = errors.Register(ModuleName, 1101, "funder with address %v does not exist")
+	ErrFundsTooLow                       = errors.Register(ModuleName, 1102, "can not defund zero coins")
+	ErrMinAmountPerBundle                = errors.Register(ModuleName, 1103, "minimum amount per bundle of coin not reached")
+	ErrMinFundingAmount                  = errors.Register(ModuleName, 1104, "minimum funding amount of coin not reached")
+	ErrFundingDoesNotExist               = errors.Register(ModuleName, 1105, "funding for pool %v and funder %v does not exist")
+	ErrFundingStateDoesNotExist          = errors.Register(ModuleName, 1106, "funding state for pool %v does not exist")
+	ErrMinFundingMultiple                = errors.Register(ModuleName, 1107, "per_bundle_amount times min_funding_multiple is smaller than funded_amount")
```

Comment:
Codes of errors should not change. e.g. ErrMinFundingMultiple was 1108 and now gets changed to 1107